### PR TITLE
Update Seed Data

### PR DESCRIPTION
**Pull request by besampson0**
Resolves [CASEFLOW-2792](https://vajira.max.gov/browse/CASEFLOW-2792) | Update Seed Data for Pending-Appeal Substitution

### Description
1. Updates existing seed data for appeal substitutions to include tasks. 
2. Adds two new sanitized appeals from prod to `/db/seeds/sanitized_json/`.

### Acceptance Criteria
- [ ] Code compiles correctly
- [ ] Substitution appeals seed data includes tasks.
- [ ] New sanitized appeals are present in `/db/seeds/sanitized_json/`.

### Testing Plan
Assumes you have a 'fresh' database, may have to run `make reset`.

PART 1
1. Search for case number '54545454'. 
2. Pull up a case that has not been dispatched.
3. Case should have a number of active tasks and completed tasks.
4. Next, pull up a case that has been dispatched. 
5. Check that there are active tasks present (should be two active tasks). 

PART 2
1. Open caseflow in your code editor.
2. Navigate to `/db/seeds/sanitized_json/`.
3. Check that `appeal-126187.json` and `appeal-119577.json` are in this folder.

### CODE REVIEW

File: db/seeds/sanitized_json/appeal-119577.json [hunk 1]
```diff
@@ -0,0 +1,6374 @@
...
+      "created_at": null,
+      "updated_at": null,
+      "status": "active",
+      "status_updated_at": null,
+      "accepts_priority_pushed_cases": null,
+      "ama_only_push": false,
+      "ama_only_request": false
+    },
+    {
+      "id": 121,
```
Comment (besampson0) [inline db/seeds/sanitized_json/appeal-119577.json]: Getting the following error -

` + Creating Organization 2000000121 
	original: JudgeTeam {"participant_id"=>nil}
    creates appeals ready for substitution (FAILED - 1)
    invokes SanitizedJsonImporter for each matching file (FAILED - 2)

Failures:

  1) Seeds::SanitizedJsonSeeds#seed! creates appeals ready for substitution
     Failure/Error: expect { subject }.to_not raise_error
     
       expected no Exception, got #<ActiveRecord::RecordInvalid: Validation failed: Url has already been taken> with backtrace:
         # ./lib/helpers/sanitized_json_importer.rb:192:in `create_new_record'
         # ./lib/helpers/sanitized_json_importer.rb:134:in `import_record'
         # ./lib/helpers/sanitized_json_importer.rb:106:in `block in import_array_of'
         # ./lib/helpers/sanitized_json_importer.rb:103:in `map'
         # ./lib/helpers/sanitized_json_importer.rb:103:in `import_array_of'
         # ./lib/helpers/sanitized_json_importer.rb:67:in `block (2 levels) in import'
         # ./lib/helpers/sanitized_json_importer.rb:66:in `each'
         # ./lib/helpers/sanitized_json_importer.rb:66:in `block in import'
         # ./lib/helpers/sanitized_json_importer.rb:65:in `import'
         # ./db/seeds/sanitized_json_seeds.rb:26:in `block in import_json_seed_data'
         # ./db/seeds/sanitized_json_seeds.rb:24:in `each'
         # ./db/seeds/sanitized_json_seeds.rb:24:in `import_json_seed_data'
         # ./db/seeds/sanitized_json_seeds.rb:15:in `seed!'
         # ./spec/seeds/sanitized_json_seeds_spec.rb:5:in `block (3 levels) in <top (required)>'
         # ./spec/seeds/sanitized_json_seeds_spec.rb:10:in `block (4 levels) in <top (required)>'
         # ./spec/seeds/sanitized_json_seeds_spec.rb:10:in `block (3 levels) in <top (required)>'
         # /Library/Ruby/Gems/2.6.0/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:124:in `block in run'
         # /Library/Ruby/Gems/2.6.0/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:110:in `loop'
         # /Library/Ruby/Gems/2.6.0/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:110:in `run'
         # /Library/Ruby/Gems/2.6.0/gems/rspec-retry-0.6.2/lib/rspec_ext/rspec_ext.rb:12:in `run_with_retry'
         # /Library/Ruby/Gems/2.6.0/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:37:in `block (2 levels) in setup'
     # ./spec/seeds/sanitized_json_seeds_spec.rb:10:in `block (3 levels) in <top (required)>'
     # /Library/Ruby/Gems/2.6.0/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:124:in `block in run'
     # /Library/Ruby/Gems/2.6.0/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:110:in `loop'
     # /Library/Ruby/Gems/2.6.0/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:110:in `run'
     # /Library/Ruby/Gems/2.6.0/gems/rspec-retry-0.6.2/lib/rspec_ext/rspec_ext.rb:12:in `run_with_retry'
     # /Library/Ruby/Gems/2.6.0/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:37:in `block (2 levels) in setup'
`

Comment (n1zyy) [reply db/seeds/sanitized_json/appeal-119577.json]: This is interesting to me. `find_existing_record` seems to know to use `url`:
https://github.com/department-of-veterans-affairs/caseflow/blob/master/lib/helpers/sanitized_json_configuration.rb#L343-L345

And Organization already uses `track_imported_ids`.
https://github.com/department-of-veterans-affairs/caseflow/blob/master/lib/helpers/sanitized_json_configuration.rb#L135-L142

This is truly a "duplicate" in that it's in two JSON files, but we import them serially so it doesn't seem like it should matter compared to them already existing in the DB.

In other words, AFAICT, this error we're getting shouldn't be happening. 😕

Comment (yoomlam) [reply db/seeds/sanitized_json/appeal-119577.json]: `default scope` bites us in the a** again!
Pushing fix

**Review by pbradin — APPROVED** [review]
Looks good, tasks are there as expected, substitutions work.
